src/swagger/user-profile-settings.swagger.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @swagger
+ * definitions:
+ *   Edit Account Profile Settings:
+ *     type: object
+ *     properties:
+ *       gender:
+ *         type: string
+ *       birthDate:
+ *         type: string
+ *       preferredLanguage:
+ *         type: string
+ *       preferredCurrency:
+ *         type: string
+ *       residence:
+ *         type: string
+ *       department:
+ *         type: string
+ *       lineManager:
+ *         type: string
+ */
+
+/**
+ * @swagger
+ * /api/view-profile:
+ *   get:
+ *     tags:
+ *       - Account Profile
```

Comment:
I think profile setting is related to users. so it should be grouped with user-related routes.